osquery/tables/system/linux/processes.cpp

```diff
@@ -13,8 +13,13 @@
 
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 
+#include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string/trim.hpp>
+#include <boost/regex.hpp>
+#include <boost/lexical_cast.hpp>
```

Comment:
If you decide to use `std::to_string` there's no longer need for `lexical_cast`.